src/main/java/org/spongepowered/common/relocate/co/aikar/timings/TimingsExport.java

```diff
@@ -293,14 +293,17 @@ public void run() {
         String timingsURL = null;
         try {
             String hostname = "localhost";
-            try {
-                hostname = InetAddress.getLocalHost().getHostName();
-            } catch (IOException e) {
-                SpongeImpl.getLogger().warn("Could not get own server hostname when uploading timings - falling back to 'localhost'", e);
+            if (!TimingsManager.privacy) {
+                try {
+                    hostname = InetAddress.getLocalHost().getHostName();
+                } catch (IOException e) {
+                    SpongeImpl.getLogger().warn("Could not get own server hostname when uploading timings - falling back to 'localhost'", e);
+                }
             }
-            HttpURLConnection con = (HttpURLConnection) new URL("http://timings.aikar.co/post").openConnection();
+            HttpURLConnection con = (HttpURLConnection) new URL("https://timings.aikar.co/post").openConnection();
             con.setDoOutput(true);
-            con.setRequestProperty("User-Agent", "Sponge/" + getServerName() + "/" + hostname);
+            String name = TimingsManager.privacy ? "" : getServerName();
+            con.setRequestProperty("User-Agent", "Sponge/" + name + "/" + hostname);
```

Comment:
@aikar Is this a valid user agent ?

Reply:
The only user agent checking on the timings side is https://github.com/aikar/timings/blob/master/lib/Security.php#L24
So it doesn't really matter what it is.

Reply:
That's just a template tho. It's not what he's using in production
https://github.com/aikar/timings/blob/c41b12ddacde256e3b54e667deca344a6587ff4f/lib/Security.php#L16